addons/core/functions/fnc_getLrSettings.sqf

```diff
@@ -86,6 +86,18 @@ if (isNil "_radioCode") then {
     _value set [TFAR_CODE_OFFSET, _radioCode];
 };
 
+private _halfDuplexOverride = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
Looking at this code again, I think the `isNil` check is actually appropriate here.

Instead of only trying to get a default value (which would indeed be more elegant using `param` with a defaultValue), this code actually checks whether the setting is **missing** and adds it as appropriate.

Were I to use `private _halfDuplexOverride = _value param [HALFDUPLEX_OVERRIDE_OFFSET, false];`, I would have no way to determine whether the value was missing from this specific LR radio or whether it was set to `false` already. Thus, I could not reliably figure out whether to actually check the config of the LR radio for the value defined there without risking overwriting a setting for a specific radio with the default value of its class.

I hope this makes any sense to you, writing it was absolutely confusing to me already 😄